pkg/sdk/informer-sync.go

```diff
@@ -80,7 +80,7 @@ func (i *informer) sync(key string) error {
 		Deleted: !exists,
 	}
 
-	sdkCtx := Context{Context: i.context}
+	sdkCtx := i.context
```

Comment:
no need to use `sdkCtx` anymore.
just do `RegisteredHandler.Handle(i.context, event)` instead.

Reply:
Done